.claude/hooks/response-mode.sh

```
@@ -0,0 +1,33 @@
+prompt=$(printf '%s' "$input" | jq -r '.prompt // empty' 2>/dev/null || printf '')
+
+# Sentinel: `$`, optional `mode`, then a mode word (case-insensitive).
+sentinel=$(printf '%s\n' "$prompt" \
+  | grep -ioE '\$[[:space:]]*(mode[[:space:]]+)?(concise|natural|default|off)' \
+  | head -1 || true)
+
+if [ -n "$sentinel" ]; then
+  value=$(printf '%s' "$sentinel" | grep -ioE '(concise|natural|default|off)' \
+    | tail -1 | tr '[:upper:]' '[:lower:]')
+  bash "$script" set "$value" >/dev/null 2>&1 || true
+  printf 'Response mode set via `%s` sentinel. Acknowledge the new mode in one short line; only treat the rest of the message as a task if it clearly contains one.\n' "$sentinel"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)